src/FrontendAccountCreation.Core.UnitTests/AccountMapperTests.cs

```diff
@@ -108,6 +108,50 @@ public void testCreateAccountModel_AsCompaniesHouseCompany_ShouldReturnAccountMo
         Assert.AreEqual("KAINOS SOFTWARE LIMITED", accountModel.Organisation.Name);
     }
 
+    [TestMethod]
+    public void CreateAccountModel_AsCompaniesHouseCompany_When_AddressIsNull_ShouldReturnAccountModelSuccessfully()
+    {
+        const string json = @"
+        {
+            ""organisation"":{
+                ""name"":""KAINOS SOFTWARE LIMITED"",
+                ""registrationNumber"":""NI019370"",
+                ""registeredOffice"":null,
+                ""organisationData"":{
+                    ""dateOfCreation"":""2023-02-23T15:27:30.681749+00:00"",
+                    ""status"":""some-status"",
+                    ""type"":""some-type""
+                }
+            }
+        }";
+
+        var organisation = JsonSerializer.Deserialize<CompaniesHouseCompany>(json, new JsonSerializerOptions(JsonSerializerDefaults.Web));
```

Comment:
Avoid creating a new 'JsonSerializerOptions' instance for every serialization operation. Cache and reuse instances instead

Reply:
Done